Misc/NEWS.d/next/Library/2017-11-26-18-48-17.bpo-32107.h2ph2K.rst

```diff
@@ -0,0 +1,4 @@
+Improve the private ``*_getnode()`` methods for UUID1 such that universally
```

Comment:
For the NEWS entry, I suggest to only describe changes on the public API. In this case, the change impacts getnode().